src/features/receipt/ReceiptContainer.tsx

```diff
@@ -126,6 +130,12 @@ export const ReceiptContainer = () => {
     }
   }, [instance, applicationMetadata]);
 
+  React.useEffect(() => {
+    if (!process?.ended) {
+      navigateToPage('confirmation');
```

Comment:
What's this for? No all apps have a confirmation task, and the ReceiptContainer should probably not render at all if the process has not ended yet?

Reply:
It is no longer needed, so I removed it :smile: